tests/__init__.py

```diff
@@ -0,0 +1,17 @@
+import json
+from django.contrib.auth.models import User
+from rest_framework.test import APITestCase
+
+from wallet.models import Category, Expense
+
+
+class TomatoTest(APITestCase):
+    def setUp(self):
+        self.user_info = {'username': 'tomato', 'password': 'top_secret'}
+        self.user = User.objects.create_user(**self.user_info)
+
+        self.sb_info = {'username': 'somebody', 'password': 'top_secret'}
+        self.sb = User.objects.create_user(**self.sb_info)
+
+        self.cname = 'category'
```

Comment:
Will category be used in **every** API test? If not, then we don't put it here.
Also, for managing database fixtures, consider [factoryboy](https://factoryboy.readthedocs.io/en/latest/orms.html#django)